js/app/cart/CartPortal.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import ReactDOM from 'react-dom';
+
+class CartPortal extends React.Component {
+  constructor(props) {
+    super(props);
+    this.el = document.querySelector('[id="bs-example-navbar-collapse-1"]');
```

Comment:
You can change the id attr of this element so it will be more explicit - change `bs-example-navbar-collapse-1` to `cart-top`